dvc/stage/cache.py

```diff
@@ -134,22 +183,13 @@ def pull(self, remote):
 
     def get_used_cache(self, used_run_cache, *args, **kwargs):
         from dvc.cache import NamedCache
-        from dvc.stage import create_stage, PipelineStage
```

Comment:
Cannot `Stage` take this responsibility of getting the `cache` entry?

Reply:
Stage class is pretty overblown as it is, didn't want to put additional functionality onto it, especially since run-cache seems to work fine separately.